pkgs/games/minetest/default.nix

```diff
@@ -29,37 +27,33 @@
 , postgresql
 , hiredis
 , libiconv
-, zlib
-, libXrandr
-, libX11
 , ninja
 , prometheus-cpp
-, mesa
```

Comment:
Did you remove mesa for both?

Reply:
Then the commit message is misleading: it says "remove mesa from buildInputs **for darwin**", suggesting it was removed for Darwin only.
Yes, removing mesa in general removes mesa for Darwin in particular.
But the whole message suggests the removal is specific for Darwin - no mention of weird hacks from upstream.

Reply:
If you actually read the commit you will see that the only `mesa` dependency was behind a Darwin conditional…